docs/tutorials/99_other/oemetadata.md

```diff
@@ -0,0 +1,102 @@
+![Contributors-1](images/Screenshots_OEMetaData/15_Contributors-1.png)
+![Contributors-2](images/Screenshots_OEMetaData/16_Contributors-2.png)
+You can also annotate the data in your table. This helps to make the data in your table comparable to others because it makes a connection between columns and/or fields to classes in the OEO. In order to do so you must add resource(s) by clicking on the `+` next to the headline `Recourses`. For all resources you must add an unique name. Ich bin etwas verwirrt und weiß nicht, ob ich das folgende richtig beschreibe!
+![Resources](images/Screenshots_OEMetaData/17_Resources.png)
+You can specify the content of a field in your table by identifying the field, giving its description and data type.
+![Field](images/Screenshots_OEMetaData/18_Field-1.png)
+The important step of making the connection between the entries in your table and the classes of the ontology happens now by providing the human readable term and the path to the OEO.
+![isabout](images/Screenshots_OEMetaData/19_isabout.png)
+**Using the OEOextended**
+If you want to use an unit which is not in the OEO yet, meaning you can't find it via the implemented search, you can click on the button `Open OEO-extended`. This will open a popup window.
+![OEOex](images/Screenshots_OEMetaData/19-1_OEOex.png)
+In the OEO-extended (OEOX) it is possible to create an additional unit using units already existing in the OEO. All created units will be reviewed and published on [GitHub](https://github.com/OpenEnergyPlatform/oeo-extended) and uploaded to the OEO. For further information on the OEOX please see [this paper](https://www.utwente.nl/en/eemcs/fois2024/resources/papers/kohler-et-al-oeox-a-post-coordination-extension-for-the-open-energy-ontology.pdf). To create the new unit you can choose your `Numerator(s)` and `Denominator(s). You can respectively choose the unit name from all units implemented in the OEO, e.g. hour, a `Unit Type` according to the power you want to have (linear, squared or cubic) and, if neccessarry, a `Unit Prefix` e.g. mega. Once you entered something in the `Unit Name`-field a fraction will appear showing you the unit you are about to create. It is possible to choose mulitple Numerator and Denominator, which will be each multiplied by each other. When you created the unit you want to use, click `Save`. Then the URI to the new unit in the OEOX will appear. You have to copy and paste it to your browser in order to entry the `Name` and `Path` in the `isAbout`-field. The OEOX is implemented on different fields in the OEMetaDataBuilder to make it easier to create new units. 
+![isabout](images/Screenshots_OEMetaData/19-2_Result.png)
+You can also provide a class of the OEO for entries of a whole column. By giving the name of the column and the name and path of the OEO class as well as the unit. Of course you can do this for multiples rows by clicking on the `+` right next to the headline `ValueReferences`.
```

Comment:
Example for is About = column name: ‘sc’ -> annotation name ‘scenario’ path ‘http://openenergy-platform.org/ontology/oeo/OEO_00000364’ 

for ‘By specifying the name of the column and the name and path of the OEO class as well as the unit’ should be changed to :
 By annotating the name of the column with the name and path of the OEO class.
 
 For value references, the text should be changed to:
 To annotate a single value from a cell as part of the current column, you can annotate the specific value, e.g. column ‘Gas’ has the value ‘co2eq’ and should be annotated with ‘Carbon Dioxide Equivalent Quantity Value’. 

The annotation can therefore only be made if values are already known. This value must be entered in the value field. The Name & Path fields are filled in in the same way as in the is About section.

This type of annotation can be very time consuming. If you have many standardised values in your column, we recommend creating an additional table that assigns the values and annotations, as this seems to make more sense.